test/index.test.ts

```diff
@@ -31,5 +31,9 @@ test('makes hotkey hint prettier', () => {
   let window = new JSDOM().window
   equal(getHotKeyHint(window, 'alt+b'), 'Alt + B')
   equal(getHotKeyHint(window, 'alt+b', { b: 'alt+b' }), 'B')
-  equal(getHotKeyHint(MAC_WINDOW, 'alt+b'), '⌥ + B')
+})
+
+test('makes mac hotkey hint prettier', () => {
+  equal(getHotKeyHint(MAC_WINDOW, 'alt+b'), '⌥ B')
+  equal(getHotKeyHint(MAC_WINDOW, 'meta+ctrl+shift+alt+b'), '⌃ ⌥ ⇧ ⌘ B')
```

Comment:
Can you also check that Win order is correct?

Reply:
Added a test for windows.